client/gutenberg/extensions/subscriptions/editor.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+//import './editor.scss';
+import edit from './edit';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import registerJetpackBlock from '../presets/jetpack/utils/register-jetpack-block';
+
+export const name = 'subscriptions';
+export const settings = {
+	title: __( 'Subscriptions' ),
+
+	description: (
+		<Fragment>
+			<p>{ __( 'Invite visitors to subscribe to your site.' ) }</p>
+		</Fragment>
+	),
+
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 208 128">
```

Comment:
lets use the 
`import renderMaterialIcon from 'gutenberg/extensions/presets/jetpack/utils/render-material-icon';`
`renderMaterialIcon` icon here. If we can